x/dualstaking/keeper/delegate_test.go

```diff
@@ -56,13 +56,6 @@ func TestDelegateFail(t *testing.T) {
 			chainID:   "mockspec",
 			amount:    -1,
 		},
-		{
```

Comment:
why did you drop this case and not check it anymore?

Reply:
dual staking module is not in charge of funds anymore so this test is pointless